src/store/models/network.spec.ts

```diff
@@ -17,6 +18,28 @@ jest.mock('utils/files', () => ({
   waitForFile: jest.fn(),
   rm: jest.fn(),
 }));
+
+jest.mock('utils/network', () => ({
+  ...jest.requireActual('utils/network'),
+  importNetworkFromZip: () => {
+    return jest.fn().mockImplementation(() => {
+      const network = {
+        id: 1,
+        nodes: {
+          bitcoin: [{}],
+          lightning: [{}],
+        },
+      };
+      return [network, {}];
+    })();
```

Comment:
This syntax feels a bit weird (having to do `()` at the end), LMK if there's anything really strange I'm doing with Jest here